### PR TITLE
Added a minimal logo as required!

### PR DESCRIPTION
**Pull request by srbhr**
Added a 'Python DS' Logo to the read me. (Font- Fira Code)

### CODE REVIEW

**Comment by prabhupant** [comment]
Hey @srbhr 

For adding any static content to readme files, the best practice is to upload the content (in this case, image) to the repo itself and then refer to it in the readme. And the link in the readme is pointing to your local machine. Could you please make the changes?

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
-# Python Data Structures
+# <img src="D:\Downloads\Python DS_1 (2).png" style="zoom:60%;" /> Python Data Structures
```
Comment (prabhupant) [inline README.md]: Please make the change here

**Comment by srbhr** [comment]
Updated the PR with the Link to the Logo, Logo is indie the logo folder.

**Comment by prabhupant** [comment]
@srbhr I have reverted merging this PR. Please make the text on the logo `python-ds` (everything in small and with a hyphen in between python and ds)

**Comment by srbhr** [comment]
Okay, before merging, I'll show you the logo!

**Comment by prabhupant** [comment]
@srbhr This looks great! Please file the PR :+1:

**Pull request merged**